lib/modules/apostrophe-login/index.js

```diff
@@ -296,12 +297,13 @@ module.exports = {
               return callback(null, false);
             }, 1000);
           }
-          return callback(err, user);
```

Comment:
`err` is definitely null at this point (see lines 294 through 299), which was unclear because I passed it to the old callback rather than passing `(null, user)`, but let's not compound that by passing it on to the new `disableIfInactive` method.

Reply:
Done